src/main/java/com/aws/greengrass/util/platforms/Platform.java

```diff
@@ -93,6 +93,8 @@ public static synchronized Platform getInstance() {
 
     public abstract SystemResourceController getSystemResourceController();
 
+    public abstract Exec getNewProcessRunner();
```

Comment:
nit: this should be `createNewProcessRunner()` - we have some poorly named methods in this class (my fault)